centreon/src/Core/PerformanceGraph/Infrastructure/Repository/DbReadPerformanceGraphRepository.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * Copyright 2005 - 2023 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+declare(strict_types=1);
+
+namespace Core\PerformanceGraph\Infrastructure\Repository;
+
+use Centreon\Infrastructure\DatabaseConnection;
+use Core\Common\Infrastructure\Repository\AbstractRepositoryRDB;
+use Core\PerformanceGraph\Application\Repository\ReadPerformanceGraphRepositoryInterface;
+
+class DbReadPerformanceGraphRepository extends AbstractRepositoryRDB implements ReadPerformanceGraphRepositoryInterface
+{
+    /**
+     * @param DatabaseConnection $db
+     */
+    public function __construct(DatabaseConnection $db)
+    {
+        $this->db = $db;
+    }
+
+    public function exists(int $performanceGraphId): bool
+    {
+        $request = $this->translateDbName('SELECT * FROM `:db`.giv_graphs_template WHERE graph_id = :id');
+        $statement = $this->db->prepare($request);
+        $statement->bindValue(':id', $performanceGraphId, \PDO::PARAM_INT);
+        $statement->execute();
+        return (bool) $statement->fetchColumn();
```

Comment:
php-cs-fixer adds a line feed before `return`
```suggestion

        return (bool) $statement->fetchColumn();
```